<launch>
  <!-- common parameters-->
  <arg name="TriggerMode_default" value="On"/>
  <arg name="TriggerSource_default" value="Software"/>    

  <node name="aravis_camera_node" pkg="camera_aravis" type="camnode">
    <param name="TriggerMode" value="$(arg TriggerMode_default)"/>
    <param name="TriggerSource" value="$(arg TriggerSource_default)"/>
  </node>
</launch>
